company/product-vision.md

```diff
@@ -2,11 +2,22 @@
 
 ## Background
 
-Our [Company Strategy](strategy.md) sets out 4 pillars that our product work is focussed around
+Our [Company Strategy](strategy.md) sets out 4 pillars that our product work is
+focussed around. Therefore our product activities should be related back to at
+least one of these 4 pillars. In addition to these pillars we will still do the
+work needed to ensure a stable, manageable, platform with a high quality
+codebase.
 
-Therefore our product activities should be related back to at least one of these 4 pillars.
+## Principles
 
-IN addition to these pillars we will still do the work needed to ensure a stable, manageable,  platform with a high quality codebase.
+#### Convention over configuration
+
+Default to picking sane default to standardize the experience for customers and
+users. This enables everyone to get the most out of FlowForge. FlowForge.com
+should always run the default configuration to ensure the best experience across
+the board.
```

Comment:
There will always be pieces of configuration that are required and cannot be defaulted. For example, the Email SMTP configuration details will be instance specific. So to say ff.cloud should always run the default configuration doesn't necessary hold true. Unless we mean it should be the default config apart from the bits that cannot be defaulted.